e2e/pod-windows-template.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ${POD_NAME}
+  labels:
+    env: test
+spec:
+  containers:
+  - name: ${POD_NAME}
+    image: mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
could we also support WS2022 and WS2022-gen2? We may not do it in this PR

Reply:
I think we can. Create a new task to do this.